util/compression.h

```diff
@@ -133,12 +133,129 @@ class ZSTDUncompressCachedData {
 
 namespace rocksdb {
 
-// Instantiate this class and pass it to the uncompression API below
+// Holds dictionary and related data, like ZSTD's digested dictionary.
+// Can be used without calling `Init()`, in which case it'll behave as if it
```

Comment:
Not a big thing, but it might be better to require that Init() be called before using it, even if calling it with an empty dictionary.